src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs

```diff
@@ -15,6 +15,7 @@ public class CryptoConfig
         private const string AssemblyName_Csp = "System.Security.Cryptography.Csp";
         private const string AssemblyName_Pkcs = "System.Security.Cryptography.Pkcs";
         private const string AssemblyName_X509Certificates = "System.Security.Cryptography.X509Certificates";
```

Comment:
sorry for changing mind twice on this already - let's revert this change for now in favor of workaround from https://github.com/dotnet/corefx/pull/16545 - I think that PR will be merged faster than this.